warp-academy-ardit/final/app/src/components/MessagesList.vue

```diff
@@ -0,0 +1,132 @@
+<script setup>
+import { useContractStore } from '../stores/contract';
+import TheLoader from './TheLoader.vue';
+const store = useContractStore();
+
+const addInteraction = async (functionType, msg) => {
+  await store.voteInteraction(functionType, msg);
+  store.getContract();
+};
+</script>
+
+<template>
+  <section>
+    <div v-if="store.contractState.messages" class="content">
```

Comment:
> right now it is not entirely visible that the list is scrollable, maybe we could display the scroll from the begining and make it a bit brighter? but i leave it up to you, it can also stay as it is

It happens only on macOs. From now, scrollbar is always visible on each system